default/scripting/species/SP_TRITH.focs.txt

```diff
@@ -27,12 +27,12 @@ Species
         [[AVERAGE_SUPPLY]]
         [[AVERAGE_DEFENSE_TROOPS]]
 
-        [[GREAT_DETECTION]]
         [[GOOD_STEALTH]]
 
         [[XENOPHOBIC_SELF]]
-
         [[XENOPHOBIC_OTHER(TRITH)]]
+        [[TELEPATHIC_DETECTION(5)]]
```

Comment:
@MatGB: The "5" which is passed as a parameter to the TELEPATHIC_DETECTION macro here is what the "@1@" gets replaced with.